src/app/pages/item/ItemList.tsx

```diff
@@ -1,36 +1,27 @@
 import React from "react"
-import PropTypes from "prop-types"
 import { connect } from "react-redux"
-import { refreshList } from "./Item.actions"
+import { refreshList } from "../../redux/Item/Item.actions"
+import { ItemProps, Item } from "../../redux/Item/Item.types"
 
-const ItemList = ({ items, refreshList }) => {
+const ItemList = ({ items, refreshList }: ItemProps) => {
     console.log(items)
     return (
         <div>
             <button onClick={refreshList}>Refresh</button>
             <ul>
-                {items.map((item) => (
+                {items.map((item: Item) => (
                     <li key={item.id}>{item.name}</li>
                 ))}
             </ul>
         </div>
     )
 }
 
-const mapStateToProps = (state) => ({
+const mapStateToProps = (state: any) => ({
     items: state.items.items,
 })
 
-const mapDispatchToProps = (dispatch) => ({
+const mapDispatchToProps = (dispatch: any) => ({
```

Comment:
You can use types
Check: https://stackoverflow.com/questions/54844839/typescript-how-to-type-the-dispatch-in-redux

Alternatively, you can use useDispatch() of React Hocks and avoid returning whole function with dispatch
https://levelup.gitconnected.com/react-redux-hooks-useselector-and-usedispatch-f7d8c7f75cdd